packages/app/src/components/Visualization/Visualization.js

```diff
@@ -27,10 +31,20 @@ export class Visualization extends Component {
         }
     }
 
-    onError = err => {
-        const error =
-            (err && err.message) ||
-            i18n.t('Error generating chart, please try again')
+    onError = response => {
+        let error
+        if (response) {
+            if (
+                response.message ==
+                'Assigned categories can only be specified together with data elements, not indicators or reporting rates'
```

Comment:
This will be updated when backend supports a custom error code for this particular error